x-pack/plugins/lens/public/indexpattern_datasource/dimension_panel/filtering.tsx

```diff
@@ -65,23 +66,23 @@ export function Filtering({
   helpMessage: string | null;
 }) {
   const inputFilter = selectedColumn.filter;
-  const [queryInput, setQueryInput] = useState(inputFilter ?? defaultFilter);
+  const { inputValue: queryInput, handleInputChange: setQueryInput } = useDebouncedValue<Query>({
+    value: inputFilter ?? defaultFilter,
+    onChange: (query) => {
```

Comment:
Due to the hook implementation this has to be wrapped with a stable pointer (`useCallback`) to prevent frequent updates

Reply:
Fixed